puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_plugin_matrix.yaml

```diff
@@ -60,7 +60,12 @@
           values:
             - fast
     execution-strategy:
-      combination-filter: "!(foreman_branch == 'develop' && ruby == '1.9.3')"
+      combination-filter: >
+        "(ruby == '1.9.3' && database == 'postgresql') ||
+        (ruby == '2.0.0' && database == 'postgresql') ||
+        (ruby == '2.1' && database == 'postgresql') ||
+        (ruby == '2.2' && database == 'postgresql') ||
```

Comment:
Optimisation - lines 64-67 can simply be written as `database == 'postgresql'`, it'll match any Ruby version.